.changeset/blue-ants-scream.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-graph': minor
+---
+
+Refactor `props` of CatalogGraphPage component. Move properties from `initialState` to `props` and use from `EntityRelationsGraphProps`.
```

Comment:
I think that you might want to elaborate that it's a breaking change here, and provide some examples for how to migrate to the new version if using the `initialState` version as it is today.

`**BREAKING**: ...` is normally how we prefix these changes too.